common/config/rush/pnpm-lock.yaml

```diff
@@ -4963,6 +5012,10 @@ packages:
       is-extglob: 2.1.1
     dev: false
 
+  /is-hexadecimal/1.0.4:
```

Comment:
why is this needed? same question for most of the deps being added here.

Reply:
I believe these are added by the new plugin `eslint-plugin-markdown`